modules/shared/components/atoms/checkbox/index.tsx

```diff
@@ -0,0 +1,88 @@
+import React, { useState } from 'react';
+import type { FC, ReactElement } from 'react';
+import classNames from 'classnames';
+import type { ICheckbox } from './ICheckbox';
+import CheckBoxSvg from './CheckBoxSvg';
+
+const CHECKBOX = {
+  SIZE: {
+    DEFAULT: 'Default',
+    SMALL: 'Small',
+  },
+};
+
+const Checkbox: FC<ICheckbox.ICheckBoxProps> = ({
+  disabled = false,
+  size = 'Default',
+  onClick,
+}): ReactElement => {
+  const [isChecked, setIsChecked] = useState(false);
+
+  const checkBoxSvgHeight = size === CHECKBOX.SIZE.DEFAULT ? '9' : '7';
+  const checkBoxSvgWidth = size === CHECKBOX.SIZE.DEFAULT ? '11' : '8';
```

Comment:
utilize the `classNames` function and move these inline conditions to it and consider changing it's name to something else such as `conditionalProperties` instead of `classNames`